gf-webapp-admin/src/component/rooms/room/longDescription/longDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+import {IRoom} from "../../../../model/room";
+import {saveRoom} from "../../../../service/room";
+import {EditBtn} from "../editBtn/EditBtn";
+
+export const LongDescription = ({room}: {room:IRoom}) => {
+  const [desc, setDesc] = useState(String(room.descriptionLong));
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  const saveHandler = () => {
+    setIsDisabled(true);
+    room.descriptionLong = desc;
+    saveRoom(room);
+  };
+
+  const disabledHandler = () => {
+    setIsDisabled(false);
+  };
```

Comment:
I've seen this somewhere before, looks like a duplicate

Reply:
I don't know how to solve it